tests/integration/test_onprem_download.py

```diff
@@ -88,14 +60,11 @@ def get_sample_granules(granules, sample_size, max_granule_size):
 
 
 def supported_collection(data_links):
-    for url in data_links:
-        if "podaac-tools.jpl.nasa.gov/drive" in url:
-            return False
-    return True
+    return all("podaac-tools.jpl.nasa.gov/drive" not in url for url in data_links)
```

Comment:
Down the road I think we should create a regex dictionary with servers we know work without custom auth loops etc, related to this when we use `S3://granule-url` users have to tell this is PODAC, maybe the same approach will work to get the credentials from whatever DAAC or mission endpoint controls access for the URL.